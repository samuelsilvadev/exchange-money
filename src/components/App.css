*,
*::before,
*::after {
	box-sizing: border-box;
}

html {
	font-size: 62.5%;
}

body {
	margin: 0;
	padding: 0;
	font-size: 1.6rem;
	line-height: 1.4;
	font-family: 'Open Sans', sans-serif;
}
