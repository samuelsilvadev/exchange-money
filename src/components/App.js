import React from 'react';

import { PocketsProvider } from '../providers/pockets/PocketsProvider';
import Pockets from './pockets/Pockets';

import styles from './App.module.css';

import './index.css';

const App = () => {
	return (
		<PocketsProvider>
			<main className={styles.wrapper}>
				<Pockets className={styles.pocketsWrapper} />
				<section className={styles.exchangeMoneyWrapper} />
			</main>
		</PocketsProvider>
	);
};

export default App;
