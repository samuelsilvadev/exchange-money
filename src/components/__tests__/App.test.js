import React from 'react';
import { render } from '@testing-library/react';

import App from '../App';

describe('<App />', () => {
	it('should render correctly', () => {
		const { container, getByText, getByLabelText } = render(<App />);

		expect(getByText('Pockets')).toBeVisible();
		expect(getByLabelText('Pocket')).toBeVisible();
		expect(container.firstChild).toMatchSnapshot();
	});
});
