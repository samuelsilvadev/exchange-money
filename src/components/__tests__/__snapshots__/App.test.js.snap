// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<main
  class="wrapper"
>
  <section
    class="pockets pocketsWrapper"
  >
    <h1
      class="title"
    >
      Pockets
    </h1>
    <div
      class="selectWrapper pocketSelect"
    >
      <label
        class="selectLabel"
        for="pocket"
      >
        Pocket
      </label>
      <div
        class="selectIconWrapper"
      >
        <select
          class="select"
          id="pocket"
        >
          <option
            value="USD"
          >
            USD
          </option>
          <option
            value="GBP"
          >
            GBP
          </option>
          <option
            value="EUR"
          >
            EUR
          </option>
        </select>
        <svg
          aria-hidden="true"
          class="selectIcon"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M12 15l-4.243-4.243 1.415-1.414L12 12.172l2.828-2.829 1.415 1.414z"
          />
        </svg>
      </div>
    </div>
    <div>
      <p
        class="pocketBallance"
      >
        1000
      </p>
      <p
        class="pocketBallanceDescription"
      >
        Total Ballance
      </p>
    </div>
  </section>
  <section
    class="exchangeMoneyWrapper"
  />
</main>
`;
